tket/src/Gate/Gate.cpp

```diff
@@ -356,6 +359,17 @@ std::optional<double> Gate::is_identity() const {
       } else
         return notid;
     }
+    case OpType::TK2: {
+      int phase = 0;
+      for (const Expr& a : params) {
+        if (equiv_0(a + 2, 4)) {
+          phase ^= 1;
```

Comment:
You could use `bool pi_phase` instead of `int phase`.

Reply:
Done.